app/views/confirmables/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "confirmables/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
```

Comment:
- [ ] Please style the register and log in forms according to the wireframe design. Firstly, center the form, then increase the width of the form inputs 👍 
### Wireframe Design
![image](https://github.com/unorjikingsley/ROR-BlogApp/assets/46839548/fc2ede73-6476-4881-b934-21a9ae9e1094)

### Your Design
![image](https://github.com/unorjikingsley/ROR-BlogApp/assets/46839548/ddf9a9f3-479c-4a0d-a6fc-86407f534281)